packages/twenty-emails/src/emails/send-email-verification-link.email.tsx

```diff
@@ -9,22 +9,33 @@ import { type APP_LOCALES } from 'twenty-shared/translations';
 type SendEmailVerificationLinkEmailProps = {
   link: string;
   locale: keyof typeof APP_LOCALES;
+  isEmailUpdate?: boolean;
 };
 
 export const SendEmailVerificationLinkEmail = ({
   link,
   locale,
+  isEmailUpdate = false,
 }: SendEmailVerificationLinkEmailProps) => {
   const i18n = createI18nInstance(locale);
+  const title = isEmailUpdate
+    ? i18n._('Confirm your new email address')
+    : i18n._('Confirm your email address');
+  const bodyId = isEmailUpdate
+    ? 'We received a request to change the email address associated with your Twenty account. Click below to confirm this change.'
+    : 'Thanks for registering for an account on Twenty! Before we get started, we just need to confirm that this is you. Click below to verify your email address.';
```

Comment:
I'm not sure if this will work, does it? The string need to be detect by Lingui during extract. I don't think it would here.